src/workerd/api/worker-rpc.c++

```diff
@@ -54,7 +54,8 @@ kj::Promise<capnp::Response<rpc::JsRpcTarget::CallResults>> WorkerRpc::sendWorke
     auto ser = serializeV8(js, js.arr(argv.asPtr()));
     // TODO(soon): We will drop this requirement once we support streaming.
     JSG_ASSERT(ser.size() < MAX_JS_RPC_MESSAGE_SIZE, Error,
-        "Serialized RPC request is too large: ", ser.size(), " <= ", MAX_JS_RPC_MESSAGE_SIZE);
+        "Serialized RPC requests are limited to 1MiB, but the size of this request was: ",
+        ser.size(), " bytes.");
```

Comment:
Probably not important, but just happened to notice that the message implies "<=" while the assert is checking "<".

Reply:
No this was a good catch, thanks Joe!